src/tools/pub/pubspec.md

```diff
@@ -272,6 +278,50 @@ publish_to: none
 {% endprettify %}
 
 
+### False_secrets
+
+When you try to [publish a package][],
+pub conducts a search for potential leaks of
+secret credentials, API keys, or cryptographic keys.
+If pub detects a potential leak in a file that would be published,
+then pub warns you and refuses to publish the package.
+
+Leak detection isn't perfect.
+To avoid false positives,
+you can tell pub not to search for leaks in certain files,
+using [`gitignore` patterns][] under
```

Comment:
I suggest changing:

"you can tell pub not to search for leaks in certain files,
using [`gitignore` patterns][]"

To:

"you can tell pub not to search for leaks in certain files,
by creating an allowlist
using [`gitignore` patterns][]"

Reply:
@jonasfj WDYT?

Reply:
I'll send a separate PR for that potential change

Reply:
https://github.com/dart-lang/site-www/pull/3704